config/routes.rb

```diff
@@ -3,7 +3,18 @@
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+  root 'questions#index'
```

Comment:
General good practice: put the `root` path at the bottom, since the route file runs top-to-bottom in terms of hierarchy.